sdk/communication/communication-administration/README.md

```diff
@@ -1,6 +1,8 @@
-# Azure Communication Administration client library for JavaScript
+# Azure Communication Services Administration client library for JavaScript
 
-The Azure Communication Administration library lets the developer create/delete users and issue tokens for Communication Services. Users and tokens can then be used when adding Chat or Calling to an app.
+The administration package is used for managing users and tokens for Azure Communication Services. This package also provides capabilities for Phone Number Administration.
```

Comment:
We shouldn't use the word "package" but "library" instead. The package is just the release artifact after building and bundling the library.

See Azure terminology: https://azure.github.io/azure-sdk/general_terminology.html